nextcord/client.py

```diff
@@ -248,6 +248,10 @@ class Client:
         Warning: While enabling this will prevent "ghost" commands on guilds with removed code references, rolling out
         to ALL guilds with anything other than a very small bot will likely cause it to get rate limited.
 
+    rollout_guild_ids: `Optional[:class:`list`[:class:`int`]]`
```

Comment:
```suggestion
    rollout_guild_ids: Optional[List[:class:`int`]]
```
for consistency and also as the docs render like this ![](https://cdn.tooty.xyz/QTNt)